modules/mod_acl_user_groups/tests/mod_acl_user_groups_tests.erl

```diff
@@ -0,0 +1,30 @@
+%% @doc Tests for mod_acl_user_groups
+-module(mod_acl_user_groups_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+-include("zotonic.hrl").
+
+person_can_edit_own_resource_test() ->
+    Context = z_context:new(testsandboxdb),
+    z_module_manager:activate(mod_content_groups, Context),
+    z_module_manager:activate(mod_acl_user_groups, Context),
+
+    %% Person must be able to edit person category
+    m_acl_rule:replace_managed(
+        [
+            {rsc, [
+                {acl_user_group_id, acl_user_group_members},
+                {actions, [view, update]},
+                {is_owner, true},
+                {category_id, person}
+            ]}
+        ],
+        testsandboxdb,
+        z_acl:sudo(Context)
+    ),
```

Comment:
This fails on Travis with:

```
11:46:36.510 [error] [testsandboxdb] z_db error {error,error,<<"42P01">>,<<"relation \"acl_rule_rsc\" does not exist">>,[{position,<<"13">>}]} in query "DELETE FROM acl_rule_rsc WHERE managed_by = $1" with [testsandboxdb]
mod_acl_user_groups_tests: person_can_edit_own_resource_test (module 'mod_acl_user_groups_tests')...*failed*
in function z_db:'-q/4-fun-1-'/5 (src/db/z_db.erl, line 289)
in call from timer:tc/2 (timer.erl, line 181)
in call from z_db:with_connection/3 (src/db/z_db.erl, line 206)
in call from m_acl_rule:delete_managed/2 (modules/mod_acl_user_groups/models/m_acl_rule.erl, line 516)
in call from m_acl_rule:replace_managed/3 (modules/mod_acl_user_groups/models/m_acl_rule.erl, line 264)
in call from mod_acl_user_groups_tests:person_can_edit_own_resource_test/0 (modules/mod_acl_user_groups/tests/mod_acl_user_groups_tests.erl, line 13)
**throw:{error,{error,error,<<"42P01">>,
              <<"relation \"acl_rule_rsc\" does not exist">>,
              [{position,<<"13">>}]}}
  output:<<"">>
```

@mworrell This looks like a timing issue, where the ACL module hasn’t been (completely) started yet. Is this correct? If so, is there a way to wait for the module to be completely initialised?